tools/benchmark.sh

```diff
@@ -205,19 +291,31 @@ function run_bulkload {
        --allow_concurrent_memtable_write=false \
        --disable_wal=1 \
        --seed=$( date +%s ) \
-       2>&1 | tee -a $output_dir/benchmark_bulkload_fillrandom.log"
-  echo $cmd | tee $output_dir/benchmark_bulkload_fillrandom.log
+       2>&1 | tee -a $log_file_name"
+  if [[ "$job_id" != "" ]]; then
+    echo "Job ID: ${job_id}" > $log_file_name
+    echo $cmd | tee -a $log_file_name
+  else
+    echo $cmd | tee $log_file_name
+  fi
```

Comment:
This same comment applies in a bunch of places, but I am not sure what the purpose of the "job_id" is.  I can see we write it to the log_file_name, but if two benchmark.sh are running concurrently, won't they overwrite the same files?  Why not make the job_id part of the output file (log_file_name) if it is specified and not write it into the log file? 

And if we are doing that, it might be helpful to always set the job_id.  If one is not specified, make it the PID of the process.  And echo that ID to the output when the benchmark.sh starts.

Reply:
So the Job ID is there, so that I can connect the output from `benchmark.sh` to a specific invocation (i.e. Job):

1. The lines in the TSV to a specific job... Note that the TSV file is typically appended to rather than being overwritten. So multiple Job ID **might** appear in the same TSV file.
2. The output log file (if needed in the case of additional information) to a specific job.

For both of the above files, the invoking job can choose where they are written to if need be by setting `OUTPUT_DIR`.

I could encode the id in the filename, but I did not need it as it can be encoded in the OUTPUT_DIR if necessary.

I also did not enforce the Job ID, as it is not something that necessarily everyone needs.

> but if two benchmark.sh are running concurrently, won't they overwrite the same files?

1. If they have not set the `OUTPUT_DIR`, then they would both overwrite the log, but the will both append to the TSV. If you want them separated, just set the `OUTPUT_DIR`.
2. Wouldn't running two benchmarks in parallel defeat the point of running a benchmark, as they would interfere with each others performance in unpredictable ways...